Gruvo/Gruvo/Controllers/AuthController.cs

```diff
@@ -18,67 +15,70 @@ public class AuthController : ControllerBase
         [HttpPost, Route("login")]
         public IActionResult Login([FromBody]UserLoginModel user)
         {
-            try
+            if (user == null)
             {
-                if (user == null)
-                {
-                    return BadRequest();
-                }
-
-                UserInfo userFromDB = AccessDatabase.MsSQL()
-                                                    .UserDAO
-                                                    .GetUserByEmailAndPwd(user.Email, user.Password);
-
-                if (userFromDB == null)
-                {
-                    return Unauthorized();
-                }
+                return BadRequest();
+            }
 
-                var token = TokenManager.GenerateToken(userFromDB.Id);
-                CookieOptions options = new CookieOptions();
-                options.Expires = DateTime.Now.AddDays(1);
+            UserInfo userFromDB = Store.MsSQL()
+                                                .UserDAO
+                                                .GetUserByEmailAndPwd(user.Email, user.Password);
+            
+            if (userFromDB == null)
+            {
+                return Unauthorized();
+            }
 
-                Response.Cookies.Append("gruvo_token", token, options);
+            string token = TokenManager.GenerateToken(userFromDB.Id);
+            
+            TokenUserPairs.GetInstance().GetPairs().Add(token, userFromDB);
+            Response.Cookies.Append("Gruvo", token);
 
-                TokenUserPairs.GetInstance().GetPairs().Add(token, userFromDB);
+            return Ok("Success!");
+        }
 
-                // TODO: Redirect
+        [Authorize(Policy = "GruvoCookie"), Route("logout")]
+        public IActionResult LogOut()
+        {
+            Response.Cookies.Delete("Gruvo");
 
-                return Ok(token);
-            }
-            catch
-            {               
-                return BadRequest("Something went wrong");
-            }
+            return Redirect("/");
         }
 
         [HttpPost, Route("signup")]
         public IActionResult Signup([FromBody]UserSignUpModel user)
         {
-            if (user == null)
-            {
-                return BadRequest();
-            }
-
             try
             {
-                AccessDatabase.MsSQL()
-                              .UserDAO
-                              .AddUser(user.Login, user.Password, user.Email, DateTime.Now);
+                if (user == null)
+                {
+                    return BadRequest();
+                }
+
+                try
```

Comment:
This "try" came from method above?) Let him go back to login.
One "try" is enough in a method. Also catch Exception ex